service/src/main/java/org/broadinstitute/listener/relay/http/TargetHttpResponse.java

```diff
@@ -83,6 +87,15 @@ public static TargetHttpResponse createTargetHttpResponse(
                 String headerValue = value.iterator().next();
                 responseHeaders.put(key, headerValue);
               });
+
```

Comment:
We need to remove this code. Otherwise, all responses will have these headers.

Reply:
I thikn we do need these....otherwise things don't work

Reply:
I'll try if removing works